app/src/main/java/com/vero/tasky/agenda/presentation/agenda/AgendaState.kt

```diff
@@ -0,0 +1,15 @@
+package com.vero.tasky.agenda.presentation.agenda
+
+import android.os.Parcelable
+import com.vero.tasky.agenda.domain.model.AgendaItem
+import kotlinx.parcelize.Parcelize
+import java.time.LocalDateTime
+
+@Parcelize
+data class AgendaState(
+    val userShortName: String = "",
+    val month: String = "",
+    val days: List<LocalDateTime> = emptyList(),
```

Comment:
This would make more sense to just have it as List\<LocalDate\>

You won't have the need to use the "time" part of it if I'm not mistaken